examples/getting_started/OpenBotEnv.py

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import gym
+import numpy as np
+import spear
+
+
+# Custom Env implementation for OpenBot
+class OpenBotEnv(spear.Env):
+
+    def __init__(self, config):
+    
+        assert config.SIMULATION_CONTROLLER.AGENT == "VehicleAgent"
+        assert "set_drive_torques" in config.SIMULATION_CONTROLLER.VEHICLE_AGENT.ACTION_COMPONENTS
+        assert "wheel_encoder" in config.SIMULATION_CONTROLLER.VEHICLE_AGENT.OBSERVATION_COMPONENTS
+
+        self._wheel_rotation_speeds = None
+
+        super().__init__(config)
+
+        # In this derived class, we are expecting different actions than the base spear.Env class. So
+        # we need to override self.action_space. We need to do this after calling super().__init__(...),
+        # because the base spear.Env class will set self.action_space to the lower-level actions it is
+        # expecting internally.
+        self.action_space = gym.spaces.Dict(spaces={"set_duty_cycles": gym.spaces.Box(low=-1, high=1, shape=(2,), dtype=np.float64)})
+    
+    def reset(self, reset_info=None):
+    
+        obs = super().reset(reset_info=reset_info)
+        assert "wheel_encoder" in obs.keys()
```

Comment:
It would be better if the C++ observation component was called `"wheel_rotation_speeds"` instead of `"wheel_encoder"`.

Reply:
Yes, changed this.